test/integration/README.md

```diff
@@ -0,0 +1,48 @@
+# Integration Tests
+
+This directory contains integration tests for the Pareto Security Agent. These tests verify that the agent works correctly on various Linux distributions and system configurations.                                                                                                                                                             │ │
+## Running Tests
+
+On NixOS, you can run the tests with the following command:
+
+```console
+$ nix build .#checks.x86_64-linux.firewall
```

Comment:
```suggestion
$ nix check .#checks.x86_64-linux.firewall
```


Build produces result file, check only runs it (so no heavy io)?

Reply:
➜  pareto-linux git:(add/firewall-tests) ✗ nix flake check .#checks.aarch64-darwin.firewall
error: unexpected fragment 'checks.aarch64-darwin.firewall' in flake reference '.#checks.aarch64-darwin.firewall'

Reply:
You are probably right, but I don't know how to just run the check.

Reply:
nix run ?

Reply:
```
 nix run .#checks.aarch64-darwin.firewall   
error: unable to execute '/nix/store/hiv0rs3f2ll340z7zm0ykmhjv551nzki-vm-test-run-Two-machines-ping-each-other/bin/vm-test-run-Two-machines-ping-each-other': No such file or directory
```